code/nodes/selectModels/funcs.q

```diff
@@ -0,0 +1,40 @@
+\d .automl
+
+// Definitions of the main callable functions used in the application of .automl.selectModels
+
+// @kind function
+// @category selectModels
+// @fileoverview Remove keras models if criteria met
+// @param mdls {tab} Models which are to be applied to the dataset
+// @param tts  {dict} Feature and target data split into train and testing sets
+// @param tgt  {(num[];sym[])} numerical or symbol vector containing the target dataset
+// @return {tab} Keras model removed if needed and removal highlighted
+selectModels.targetKeras:{[mdls;tts;tgt]
+  if[1~checkimport[0];
+    :?[mdls;enlist(<>;`lib;enlist `keras);0b;()]
+    ];
+  multiCheck:`multikeras in mdls`model;
```

Comment:
This check should be a check against the form of model being used i.e. is it keras and is it multiclass rather than just for the multikeras function. Currently this wouldn't support 'bring your own keras model' for users wanting to do so